### DISCUSSION TITLE
wine: How to set different configurations for different applications?

### DISCUSSION THREAD

**Discussion by danielkrajnik** [netblue30/firejail#6084] [Q&A] [answered]
How would you set different configurations for different wine applications? For example application A doesn't need any access to the host system, but application B needs access to the `/mnt` folder?

My current `wine.local`:

```
whitelist ${HOME}/.wine
whitelist ${HOME}/.wine-rhino-7
whitelist ${HOME}/.wine-everything
whitelist ${HOME}/.cache/winetricks
whitelist ${HOME}/.cache/wine
whitelist ${HOME}/Desktop
# tmpfs ${HOME}/Desktop
disable-mnt
net none
# ignore allow-debuggers
```

Relates to:

* #6138
* #6826

**Comment by rusty-snake** [comment, marked as answer]
```
firejail --include=~/.config/firejail/wine-A.local --profile=wine --env=WINEPREFIX=~/.wine-A /usr/bin/wine [wine args]

firejail --include=~/.config/firejail/wine-B.local --profile=wine --env=WINEPREFIX=~/.wine-B /usr/bin/wine [wine args]
```

**Reply by danielkrajnik** [reply]
Sorry, you are right. I fixated on using the symlink in `/usr/local/bin/wine -> firejail` and forgot that you can just use firejail directly.

**Reply by danielkrajnik** [reply]
Can I ask you why you wrote `--env=WINEPREFIX=~/.wine-A` instead of `env WINEPREFIX=~/.wine-A` (`env` as in `/usr/bin/env` instead of the `--env` option)?

**Reply by rusty-snake** [reply]
No real reason. However, setting it with firejail allows you to move it to wine-*.local.
